app/src/main/res/values/strings.xml

```diff
@@ -10,16 +10,16 @@
 	<string name="text_dark">DARK</string>
 
 	<string name="about_author">Author: Paper Airplane Dev - Fung Jichun</string>
-	<string name="about_text">"Night Screen can help you prevent from the harm of strong light at night.\nSome devices\' lowest brightness are still high so we can make it lower by creating a mask on screen.\n\nIf you want to support this application, you could send a bit money to my Alipay: 316643843\@qq.com"</string>
+	<string name="about_text">"Night Screen can help protect you from the harm of strong light at night.\nSome devices\' lowest brightness is still high so we can make it lower by creating a mask on screen.\n\nIf you want to support this application, you could send some money to my Alipay: 316643843\@qq.com"</string>
 	<string name="about_other">Github Repo: <a href="https://github.com/fython/Blackbulb">https://github.com/fython/Blackbulb</a>\nMy blog: <a href="http://feng.moe">http://feng.moe</a>\nMy sina weibo: <a href="http://weibo.com/fython">\@某烧饼</a>\nLogo Designer: Google+ Youlun Zhai</string>
 
 	<string name="dialog_overlay_enable_title">Enable Overlay Status Bar</string>
-	<string name="dialog_overlay_enable_message">Enable this can make mask on the top of screen, but it may needs a permission in some devices (Such as MIUI and Android 6.0)\nIf it couldn\'t work, please allow this app to show system alert window.</string>
+	<string name="dialog_overlay_enable_message">Enabling this can mask the top of screen, but it may need a permission on some devices (such as MIUI and Android 6.0)\nIf it won\'t work on first try, please allow this app to show system alert windows.</string>
```

Comment:
"couldn\'t" was corrected to "won\'t", but should be corrected to "doesn\'t"

"on first try" should be "on the first try"